src/components/images/logo-lg.tsx

```diff
@@ -0,0 +1,54 @@
+import {HtmlHTMLAttributes} from "react";
+
+type Props = HtmlHTMLAttributes<SVGSVGElement> & {
+  height?: number
+  width?: number
+}
+
+const PressFooterLogoLg = ({height = 89, width = 579, ...props}: Props) => {
```

Comment:
You should add `{...props}` to the `svg` attributes.

Reply:
Ok.  Thanks for the feedback!